platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -660,19 +661,29 @@ private List<EventImpl> getStronglySeenInPreviousRound(final EventImpl event) {
                 lastConsensusTime = ConsensusUtils.calcMinTimestampForNextEvent(lastConsensusTime);
             }
         }
+
+        // Future work: enable the ancient mode to be configurable
+        final AncientMode ancientMode = GENERATION_THRESHOLD;
+
+        // Future work: prior to enabling a generation based ancient mode, we need to use real values for
```

Comment:
This comment says prior to enabling a `generation` based ancient mode and then talks about rounds.

Reply:
Lol, oops. Fixed.